test/integration/node/package.json

```diff
@@ -1,11 +1,9 @@
 {
   "private": true,
   "description": "chart.js should work in Node",
+  "type": "module",
   "scripts": {
-    "test": "npm run test-cjs",
-    "test-cjs": "node test.cjs",
-    "test-mjs": "node test.mjs",
-    "TODO": "test-mjs should be enambled for chart.js v4"
+    "test": "node test.js"
```

Comment:
The idea was, the package should work for both, the esm and cjs flavours of node, that's why there were 2 tests. It does not work for cjs now?

Reply:
@kurkle 

https://github.com/chartjs/Chart.js/pull/10525#issuecomment-1199256469

In this PR we dropping commonjs in favor of ESM only package. And also provides UMD bundle for usage in browsers without bundlers.

Reply:
We can continue to discuss this option. It's easy to restore cjs bundle.

Reply:
UMD works for CJS, we don't have a separate CJS bundle in v3 either.

Reply:
Note that you can use an ESM library from a CJS project by doing a dynamic import rather than static import. I think it's quite rare people would want to use Chart.js from a CJS project in the first place, so I think that's a good enough solution since it enables us to simplify things so much and have a smaller package size.

Reply:
so add a test case doing that, so we can be sure it works?

Reply:
@kurkle done